### ISSUE TITLE
Missing method to specify the arrival time when creating Packets from scratch.

### ISSUE THREAD

**Issue by axelpodehl** [aboutsip/pkts#104]
When writing data packets (read from another source, timestamped) into a PCAP file, it would be be nice to be able to set the arrival time to the time of that packet. And NOT the current system time.

In other words, I'm looking for a new method TransportPacketFactory.create(...,long arrivalTime) where I can explicitly specify the time of that packet. My processing time, is not the same as the packet's reception time.

**Comment by jonbo372** [comment]
Yep, good one. I'll see when I get to it but of course, pull requests are welcome!
